backend/tests/unit/core/test_query_subquery.py

```diff
@@ -23,7 +23,7 @@ async def test_build_subquery_filter_attribute_text(
 
     expected_query = """
     WITH n
-    MATCH path = (n)-[:HAS_ATTRIBUTE]-(i:Attribute { name: $filter1_name })-[:HAS_VALUE]-(av:AttributeValue { value: $filter1_value })
+    MATCH path = (n:Node {uuid: n.uuid})-[:HAS_ATTRIBUTE]-(i:Attribute { name: $filter1_name })-[:HAS_VALUE]-(av:AttributeValue { value: $filter1_value })
```

Comment:
I'm surprised this is required
if `n` is already defined, we shouldn't need to filter it ...